### PR TITLE
Broken open-stack api lock

### PR DESCRIPTION
**Pull request by kthare10**
Addresses https://github.com/RENCI-NRIG/orca5/issues/247

### CODE REVIEW

File: handlers/ec2/resources/handlers/ec2/handler.xml [hunk 1]
```diff
@@ -147,10 +246,6 @@
                         <env key="EC2_INSTANCE_TYPE" value="${ec2.instance.type}" />
                         <env key="EC2_PUBLIC_NETWORK" value="${ec2.public.network}" />
                         <env key="EC2_MGMT_NETWORK" value="${ec2.management.network}" />
-                        <env key="EC2_SLICE_USER_EMAIL" value="${ec2.slice.user.email}" />
```
Comment (ibaldin) [inline handlers/ec2/resources/handlers/ec2/handler.xml]: Are these not needed anymore?

File: handlers/ec2/resources/scripts/nova-essex-start [hunk 2]
```diff
@@ -67,13 +67,7 @@ try:
                              os.environ['EC2_PUBLIC_NETWORK'],
                              os.environ['PROJECT_NAME'],
                              os.environ['USER_NAME'],
-                             os.environ['EC2_SLICE_USER_EMAIL'],
```
Comment (ibaldin) [inline handlers/ec2/resources/scripts/nova-essex-start]: Same here - not needed?

Comment (kthare10) [reply handlers/ec2/resources/scripts/nova-essex-start]: No, not needed as they are being passed only for project creation in create.project macro. No longer needed for vm start

File: handlers/ec2/resources/handlers/ec2/handler.xml (hunk 1 above)
Comment (kthare10) [reply handlers/ec2/resources/handlers/ec2/handler.xml]: No, not needed as they are being passed only for project creation in create.project macro. No longer needed for vm start

**Pull request merged**